### ISSUE TITLE
Bug: Using a buffer created on Linux, on a Windows machine, crashes the application.

### ISSUE THREAD

**Issue by GlacierFox** [heyman/heynote#264]
**Describe the bug**
I save my buffer when using the app on linux to cloud storage so I can use it on other machines.
Are buffers compatible between Linux and Windows? When I open the app on my Windows machine the text on the screen sort of flashes and I can't type anything.  I then get a huge memory leak which fills up my system ram in 30 seconds then crashes the app. The app works fine if I use a fresh buffer on Windows.

**To Reproduce**
1 - Start Hello Note on Linux and save the buffer.
2 - Open that same buffer up on a Windows machine.
3 - Memory Leak.

**Expected behavior**
To use the same buffer file between machines.

**Desktop (please complete the following information):**
 - OS: RHEL
 - Version 9.4

- OS: Windows 11

**Comment by heyman** [comment]
Hi! Could you supply a buffer file that causes the crash on windows?

**Comment by GlacierFox** [comment]
Yeah sure, here's a one that causes problems for me (It's just got random code stuff in, contains nothing private) @heyman 

[buffer.txt](https://github.com/user-attachments/files/16977711/buffer.txt)
